_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: No more KubeVirt
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 178.
+
```

Comment:
To avoid making it appear that killing kubevirt is the only change: How about adding a paragraph that this cycle was about stabilization and bug fixing, and there are a lot of crash fixes? In particular, https://github.com/cockpit-project/cockpit/issues/10003 was a mild security issue, so I think it's prudent to point this out explicitly in the release note to justify updating to it.

Reply:
Agreed! I had this thought as well when looking at the rendered page.

Reply:
Fixed this in a recent edit